docs/language/operators/over.md

```diff
@@ -12,45 +12,21 @@ The `over` operator traverses complex values to create a new sequence
 of derived values (e.g., the elements of an array) and either
 (in the first form) sends the new values directly to its output or
 (in the second form) sends the values to a scoped computation as indicated
-by `<lateral>`, which may represent any Zed subquery operating on the
-derived sequence of values as `this`.
+by `<lateral>`, which may represent any Zed [subquery](../lateral-subqueries.md) operating on the
+derived sequence of values as [`this`](../dataflow-model.md#the-special-value-this).
 
 Each expression `<expr>` is evaluated in left-to-right order and derived sequences are
 generated from each such result depending on its types:
-* an array value generates each of its element,
+* an array value generates each of its elements,
 * a map value generates a sequence of records of the form `{key:<key>,value:<value>}` for each
 entry in the map, and
 * all other values generate a single value equal to itself.
 
-Records can be converted to maps with the [_flatten_ function](../functions/flatten.md)
+Records can be converted to maps with the [`flatten` function](../functions/flatten.md)
```

Comment:
> Records can be converted to maps...

Strictly speaking, it doesn't seem like `flatten` converts them to maps. Rather, it generates a sequence of values similar to what `over` does with maps, but even then, there's still the difference that `flatten` creates an array for the path of the record keys whereas `over` on a map outputs the keys as is. Does this bother anyone? It doesn't _quite_ bother me enough to insist on a major expansion of this text. However, user confusion about maps has come up a few times (e.g., the subtle difference between true Zed map types and what I've been calling "JSON-compatible pseudo-maps" (#4565)) which is why I perked up when I saw a possible oversimplified reference to "maps" here.

Anyone have reactions?